ts/screens/authentication/LandingScreen.tsx

```diff
@@ -296,133 +319,232 @@ export const LandingScreen = () => {
-            <HSpacer size={8} />
-            <NBButtonText>
-              {isCieSupported()
-                ? I18n.t("authentication.landing.loginCie")
-                : I18n.t("authentication.landing.loginSpid")}
-            </NBButtonText>
-          </ButtonDefaultOpacity>
-          <VSpacer size={16} />
-          <ButtonDefaultOpacity
+            fullWidth={true}
             accessibilityLabel={
               isCieSupported()
                 ? I18n.t("authentication.landing.loginSpid")
                 : I18n.t("authentication.landing.loginCie")
             }
-            accessibilityRole="button"
-            accessible={true}
-            style={secondButtonStyle}
-            block={true}
-            primary={true}
-            iconLeft={true}
+            color="primary"
+            // if CIE is not supported, since the new DS has not a
+            // "semi-enabled" state, we leave the button enabled
+            // but we navigate to the CIE unsupported info screen.
+            label={
+              isCieSupported()
+                ? I18n.t("authentication.landing.loginSpid")
+                : I18n.t("authentication.landing.loginCie")
+            }
+            icon={isCieSupported() ? "spid" : "cie"}
             onPress={
               isCieSupported() ? navigateToIdpSelection : navigateToCiePinScreen
             }
-            testID={
-              isCieSupported()
-                ? "landing-button-login-spid"
-                : "landing-button-login-cie"
-            }
-          >
-            <Icon
-              name={isCieSupported() ? "navProfile" : "cie"}
-              color="white"
+          />
+          <VSpacer size={BUTTON_SPACING} />
+          <View style={IOStyles.selfCenter}>
+            <ButtonLink
+              accessibilityLabel={I18n.t("authentication.landing.privacyLink")}
+              color="primary"
+              label={I18n.t("authentication.landing.privacyLink")}
+              onPress={navigateToPrivacyUrl}
             />
-            <HSpacer size={8} />
-            <NBButtonText>
-              {isCieSupported()
-                ? I18n.t("authentication.landing.loginSpid")
-                : I18n.t("authentication.landing.loginCie")}
-            </NBButtonText>
-          </ButtonDefaultOpacity>
-          <VSpacer size={16} />
-          <Link
-            style={styles.link}
-            onPress={navigateToSpidCieInformationRequest}
-          >
-            {isCieSupported()
-              ? I18n.t("authentication.landing.nospid-nocie")
-              : I18n.t("authentication.landing.nospid")}
-          </Link>
-        </View>
-      </BaseScreenComponent>
+            {insets.bottom === 0 && <VSpacer size={BUTTON_SPACING} />}
+          </View>
+        </ContentWrapper>
+      </SafeAreaView>
     );
   };
 
   // Screen displayed during the async loading of the JailMonkey.isJailBroken()
-  const renderLoadingScreen = () => (
+  const LoadingScreen = () => (
```

Comment:
In the demo, you can see from the video [Small.Android.Device.+.Jailbroken.mov](https://private-user-images.githubusercontent.com/16268789/307821723-6cfb9af4-05c5-4a59-b213-a31ff9f2010c.mov?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkxMjk4NDgsIm5iZiI6MTcwOTEyOTU0OCwicGF0aCI6Ii8xNjI2ODc4OS8zMDc4MjE3MjMtNmNmYjlhZjQtMDVjNS00YTU5LWIyMTMtYTMxZmY5ZjIwMTBjLm1vdj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI4VDE0MTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTljNjExZTlhMGMyNWI4YTA1ODRlMTcxYjcyMGI3YWYzYjMzNGI4MmJiNjZiYThmNzIzN2ZkODNmYzU3ODVmNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RgcF2--H396nLx3SpUN4XTysN83YxWIr8PE2Jt0JWQk) that there are two spinners. One of them is this one. How about we take it out or try to make an adaptation?

Reply:
I'l check it out!

Reply:
I checked it out, and found that the double spinners are all inside the `RootedDeviceModal` because it is wrapped inside a `LoadingSpinnerOverlay`. As mention above, we'll address the adoption of the new DS to this screen in its own JIRA story, so I suggest to take care of this in its dedicated PR.

Reply:
Yes, for me it's better too